examples/k8s/README.md

```diff
@@ -1,19 +0,0 @@
-# Example Kubernetes Configurations
```

Comment:
Do we want to link to the new location for these examples in case folks come looking for them?